crowbar_framework/app/models/provisioner_service.rb

```diff
@@ -51,6 +51,14 @@ def create_proposal
   end
 
   def validate_proposal_after_save proposal
+    proposal["attributes"]["provisioner"]["packages"].each do |platform, packages|
+      packages.each do |package|
+        unless package =~ /(^\w[\w\+\.<=>-]+$)/
```

Comment:
What do you think of this instead?
`(^\w[\w\+\.-]?[<>]?=?[\w\+\.-]+$)`
This prevents wrong usage of the version operators. Only downside I see is that package names now must consist of at least two characters.